### PR TITLE
en, zh: add version relation between dm and tidb

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Add the corresponding relationship between TiDB and DM

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/overview.md [hunk 1]
```diff
@@ -5,7 +5,11 @@ category: reference
 
 # Data Migration 简介
 
-[DM](https://github.com/pingcap/dm) (Data Migration) 是一体化的数据同步任务管理平台，支持从 MySQL 或 MariaDB 到 TiDB 的全量数据迁移和增量数据同步。使用 DM 工具有利于简化错误处理流程，降低运维成本。
+[TiDB Data Migration](https://github.com/pingcap/dm) (DM) 是一体化的数据同步任务管理平台，支持从 MySQL 或 MariaDB 到 TiDB 的全量数据迁移和增量数据同步。使用 DM 工具有利于简化错误处理流程，降低运维成本。
+
+> **注意：**
+>
+> DM 以 SQL 语句的形式将数据同步到 TiDB 中，因此 DM 兼容所有版本的 TiDB。推荐使用 DM 的最新已发布版本 v1.0.4。下载方式参见 [DM 下载链接](https://pingcap.com/docs-cn/stable/reference/tools/download/#tidb-dm-data-migration)。
```
Comment (yikeke) [inline zh/overview.md]:
```suggestion
> DM 以 SQL 语句的形式将数据同步到 TiDB 中，因此各个版本的 DM 都分别兼容**所有版本**的 TiDB。在生产环境中，推荐使用 DM 的最新已发布版本。下载方式参见 [DM 下载链接](https://pingcap.com/docs-cn/stable/reference/tools/download/#tidb-dm-data-migration)。
```
建议不要给出特殊信息“v1.0.4”，免得每次出新版本都要来修改这个信息？

Comment (lilin90) [reply zh/overview.md]: @WangXiangUSTC 看下是否可以建议用户用最新已发布版本？

Comment (WangXiangUSTC) [reply zh/overview.md]: 可以的

**Comment by lilin90** [comment]
Addressed the comment. @yikeke @WangXiangUSTC PTAL again~ Thanks!

**Pull request merged**